### ISSUE TITLE
README switch-case example does not work with webpack env object

### ISSUE THREAD

**Issue by cmavelis** [survivejs/webpack-merge#177]
Using `switch(env)` when env is an object and the cases are strings will always go to the `default`.  It seems either the example is out of date or it should just be an `if` block.

```
module.exports = env => {
  switch(env) {
    case 'development':
      return merge(commonConfig, developmentConfig);
    case 'production':
      return merge(commonConfig, productionConfig);
    default:
      throw new Error('No matching configuration was found!');
  }
}
```

**Comment by cmavelis** [comment]
This is what I used instead:
```
module.exports = env => {
    if (env['development']) {
        return merge(commonConfig(env), devConfig);
    } else if (env['production']) {
        return merge(commonConfig(env), prodConfig);
    } else {
        throw new Error('Please specify webpack "--env development" or "--env production"');
    }
};
```

**Comment by bebraw** [comment]
webpack 5 had a breaking change here but I forgot to update the readme. Thanks for the heads up.

**Comment by bebraw** [comment]
This was solved in #180.
